src/mmw/js/src/core/views.js

```diff
@@ -712,8 +712,9 @@ var MapView = Marionette.ItemView.extend({
         });
     },
 
-    createDataCatalogShape: function(geom) {
-        var geomId = geom && geom.properties.id,
+    createDataCatalogShape: function(result) {
+        var geom = result.get('geom'),
+            geomId = geom && geom.properties.id,
```

Comment:
This id is the same as `result.get('id')`. Now that we're passing the whole result around, consider removing the geojson augmentation that happens [here.](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/data_catalog/models.js#L243)

Reply:
Addressed in e2fb98d.